lib/utils/fields.js

```diff
@@ -67,3 +67,30 @@ export function dataArrayToObject(
   });
   return dataObject;
 }
+
+const DATA_OBJECT_TO_ARRAY_DEFAULT_OPTIONS = {
+  multivalueDelimiter: '; ',
+};
+export function dataObjectToArray(dataObject, fields, options = {}) {
+  const { multivalueDelimiter } = {
```

Comment:
Just curious, so this construct is a way to externalize the value of a constant rather than hardcoding it?

Reply:
It's basically this pattern: https://www.codereadability.com/what-are-javascript-options-objects/ -- except I defined the defaults outside of the function, rather than inside (just sort of a matter of preference); and used [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to merge the defaults and the user-specified options.